components/aws/sagemaker/unit_tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3
```

Comment:
Can we get the Dockerfile to use this file instead of hardcoding the values inside?

Reply:
done